meta-xt-prod-devel-rcar-control/recipes-guests/domu/files/domu-pvcamera.cfg

```diff
@@ -1,9 +1,8 @@
-
 # HDMI_IN camera configuration
-vcamera = [[ 'backend=DomD, be-alloc=0, unique-id=video0:media0, max-buffers=4',
-             'format=NV12, resolution=1024x768',
-             'format=NV12, resolution=1920x1080'
-          ]]
+# vcamera = [[ 'backend=DomD, be-alloc=0, unique-id=video0:media0, max-buffers=4',
```

Comment:
Why you are commenting code instead of removing it?

Reply:
The reason why I commented the code instead of removing it is to have all possible examples visible to the user. As it was discussed internally I will add "This is example for ХХХ use-case" above each commented block.

Reply:
"+"